app/javascript/controllers/feedback_form_controller.js

```diff
@@ -0,0 +1,123 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Behavior for the feedback modal and standalone feedback page
+export default class extends Controller {
+  static targets = [ 'agent', 'viewport', 'lastSearch' ]
+
+  // For the modal, connect is fired once the modal is opened
+  // For the standalone page, connect is fired on page load
+  connect() {
+    // For a standalone form, certain elements need to be hidden
+    this.handleStandaloneDisplay()
+    // Set hidden field values required for submission
+    this.setHiddenFieldValues()
+  }
+
+  handleStandaloneDisplay() {
+    if (this.isStandalone()) {
+      document.querySelector('div.new-link').classList.add('d-none')
+      document.querySelectorAll('button.close').forEach((button) => {
+        button.classList.add('d-none')
+      })
+    }
+  }
+
+  isStandalone() {
+    return document.querySelectorAll('div[data-blacklight-modal]').length > 0
+  }
+
+  // Attached to the "Send" button via data action for submission event turbo:submit-end
+  async submitForm(event) {
+    const response = await event.detail.fetchResponse.response.json()
+    this.displayResponse(response)
```

Comment:
This is completely bypassing the point of using turbo, which is to render the html server side.  I think this makes it more confusing that it needs to be.

Reply:
This uses turbo stream now.